src/appleseedmaya/exporters/dagnodeexporter.h

```diff
@@ -29,13 +29,18 @@
 #ifndef APPLESEED_MAYA_EXPORTERS_DAGNODEEXPORTER_H
 #define APPLESEED_MAYA_EXPORTERS_DAGNODEEXPORTER_H
 
+#pragma once
```

Comment:
If you introduce `#pragma once` (which is a great move) then you need to remove the header guards.

Reply:
So you prefer to remove the header guards? I put #pragma once there during bug hunting and forgot to take it out.

Reply:
We fully transitioned appleseed core from header guards to `#pragma once` since they are simpler, less error prone, possibly also faster and universally supported. At some point it would be nice to do the same in appleseed-maya, but not necessarily in this PR.

Reply:
Agree, I removed the #pragma once
This is better done in a separate PR and not mix more together. 
This PR is already a mix of unrelated issues.

Reply:
Agreed, perfect.